code/modules/mob/living/carbon/human/species.dm

```diff
@@ -18,96 +20,130 @@
 	var/burn_damage_icon_state = "human_burn"
 	///damage mask icon we want to use when drawing wounds
 	var/damage_mask_icon = 'icons/mob/dam_mask.dmi'
-	///If set, draws this from icobase when mob is prone.
-	var/prone_icon
 	///icon for eyes
 	var/eyes = "eyes_s"
+	///Color of the blood specific to our species
+	var/blood_color = "#A10808"
+	///Color of the gibs that spawn from our species [/mob/living/carbon/human/spawn_gibs]
+	var/flesh_color = "#FFC896"
+	///Used when setting species.
+	var/base_color
+	///If the species only has one hair color
+	var/hair_color
+	///Used in icon caching.
+	var/race_key = 0
+	///Used in icon caching.
+	var/icon/icon_template
 
-	var/datum/unarmed_attack/unarmed           // For empty hand harm-intent attack
-	var/datum/unarmed_attack/secondary_unarmed // For empty hand harm-intent attack if the first fails.
+	//----Grouped these because they get set on New()
+	///hud that our mob uses, gets given the type stored in hud_type on New()
 	var/datum/hud_data/hud
+	///type that our hud gets set to on New()
 	var/hud_type
-	var/slowdown = 0
-	var/taste_sensitivity = TASTE_NORMAL
-	var/gluttonous        // Can eat some mobs. 1 for monkeys, 2 for people.
-	var/rarity_value = 1  // Relative rarity/collector value for this species. Only used by ninja and cultists atm.
-	var/datum/unarmed_attack/unarmed_type = /datum/unarmed_attack
+	///For empty hand harm-intent attack
+	var/datum/unarmed_attack/unarmed
+	///type that our unarmed gets set to on New()
+	var/unarmed_type = /datum/unarmed_attack
+	///For empty hand harm-intent attack if the first fails.
+	var/datum/unarmed_attack/secondary_unarmed
+	///type that our secondary_unarmed gets set to on New()
 	var/secondary_unarmed_type = /datum/unarmed_attack/bite
-	var/default_language_holder = /datum/language_holder
-	var/speech_verb_override
-	var/secondary_langs = list()  // The names of secondary languages that are available to this species.
-	var/list/speech_sounds        // A list of sounds to potentially play when speaking.
-	var/list/speech_chance
-	var/has_fine_manipulation = TRUE // Can use small items.
-	var/count_human = FALSE // Does this count as a human?
```

Comment:
why are you removing this? Not all species have this as true, so using ishuman is not a correct replacement.

Reply:
ah it's false by default. My bad, I'll turn it into a flag instead